DigitalExcellence/dex-frontend
# [Bug] Add Project Modal - Ignore Background Click, Add Close Button

## Description

On the Add Project Modal, Ignoring background click to close modal (also disabling the use of the Esc button to close modal) and adding a close button in the top right.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Add Project
2. Add project manually
3. Insert info until step 3
4. Click outside of modal or hit Esc button, no longer able to close modal.
5. Click x button in top right to close modal

Photos of Project Model with Close button (desktop and mobile views)

![close-button-large](https://user-images.githubusercontent.com/21061492/138180830-be7e6b95-b6c8-41e2-961a-1ca6e128a22a.PNG)

![close-button-phone](https://user-images.githubusercontent.com/21061492/138180837-5a416ef3-295e-4065-9bdc-85ec3f1351b7.PNG)

Video of Close button working
https://user-images.githubusercontent.com/21061492/138181076-a92c421d-d832-4d55-98ec-953fdff74f4e.mp4

## Link to issue

Closes: #558
